src/compile.ml

```diff
@@ -62,6 +62,11 @@ let rec compile : bool -> Files.module_path -> unit = fun force path ->
             data.pdata_finalize ss st
       in
       ignore (List.fold_left handle sig_st (parse_file src));
+      let not_local_fst _ (sym, _) = Terms.(sym.sym_expo) <> Terms.Local in
+      let not_local _ sym = Terms.(sym.sym_expo) <> Terms.Local in
```

Comment:
should be the other way around: first define not_local, then not_local_fst using not_local to avoid code duplication